docs/user/security/fips-140-2.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[xpack-security-fips-140-2]]
+=== FIPS 140-2
+
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB 140-2), 
+titled "Security Requirements for Cryptographic Modules" is a U.S. government computer security standard
+used to approve cryptographic modules.
+
+{kib} offers a FIPS 140-2 compliant mode and as such can run in a Node.js environment configured with an FIPS 
```

Comment:
nit: change `with an FIPS` to `with a FIPS`

Reply:
++ nice catch! Fixed in latest commit